diff --git a/src/TugboatClient.php b/src/TugboatClient.php
index 90d14a5592830c79f081eb79ca143cea78862776..24a6c0d03d3461022320004c2ba29973e90e09dc 100644
--- a/src/TugboatClient.php
+++ b/src/TugboatClient.php
@@ -10,7 +10,7 @@ use Drupal\Core\Http\ClientFactory;
  */
 class TugboatClient {

-  const API_URL = 'https://api.tugboat.qa/v3/';
+  const API_URL = 'https://api.tugboatqa.com/v3/';

   protected $client;
   protected $settings;
