From 435c7e166c73356661d2b72536a7431681490caa Mon Sep 17 00:00:00 2001
From: mglaman <mglaman@2416470.no-reply.drupal.org>
Date: Mon, 16 May 2022 14:45:44 +0000
Subject: [PATCH] Use tugboatqa.com

---
 src/TugboatClient.php | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/TugboatClient.php b/src/TugboatClient.php
index 90d14a5..24a6c0d 100644
--- a/src/TugboatClient.php
+++ b/src/TugboatClient.php
@@ -10,7 +10,7 @@ use Drupal\Core\Http\ClientFactory;
  */
 class TugboatClient {

-  const API_URL = 'https://api.tugboat.qa/v3/';
+  const API_URL = 'https://api.tugboatqa.com/v3/';

   protected $client;
   protected $settings;
--
GitLab
